### PR TITLE
Solve Package Page Tooltip Accessibility Bugs and Framework Filter Checkbox List Accessibility Bug

### PR DESCRIPTION
**Pull request by Goodyear2017**
Disable Bootstrap popover JavaScript on framework filter and support framework badges, keep the popover classes and layout, add JavaScript function to make the tooltip popup, narrator can read the tooltip.

Add up and down arrow key JavaScript function to make users are able to use arrow keys to move between framework filter mode checkboxes.
Bug links:

- https://devdiv.visualstudio.com/DevDiv/_queries/edit/?tempQueryId=91c293b3-665c-4cb3-876f-e7e4c119b7d1&id=1970893
- https://devdiv.visualstudio.com/DevDiv/_queries/edit/?tempQueryId=91c293b3-665c-4cb3-876f-e7e4c119b7d1&id=1970906

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/_SupportedFrameworksBadges.cshtml [hunk 1]
```diff
@@ -84,15 +84,22 @@
 @helper DisplayFrameworkBadge(PackageFrameworkCompatibilityData tfmCompatibilityData, FrameworkContext context, int? itemIndex, string eventName)
 {
     <a href=@Url.FrameworksTab(Model.PackageId, Model.PackageVersion)
-       @if (itemIndex.HasValue)
-       {
-            @: data-track="@eventName" data-track-value="@itemIndex" data-click-source="FrameworkBadge"
-            @: data-package-id="@Model.PackageId" data-package-version="@Model.PackageVersion"
-            @: data-badge-framework="@tfmCompatibilityData.Framework.GetShortFolderName()" data-badge-is-computed="@tfmCompatibilityData.IsComputed"
+       @if (itemIndex.HasValue) { @: data-track="@eventName" data-track-value="@itemIndex" data-click-source="FrameworkBadge"
```
Comment (ryuyu) [inline src/NuGetGallery/Views/Packages/_SupportedFrameworksBadges.cshtml:87]: nit: can we adjust formatting here so it is consistent with how it was before?

Comment (Goodyear2017) [reply src/NuGetGallery/Views/Packages/_SupportedFrameworksBadges.cshtml]: Just reformatted it. Thanks!

File: src/NuGetGallery/Content/Site.css [hunk 2]
```diff
@@ -1726,3 +1726,36 @@ img.contributors-contributor-avatar {
     font-size: 1.1em;
     padding: 5px 15px 5px 15px;
 }
+/* Packages page tooltip */
```
Comment (erdembayar) [inline src/NuGetGallery/Content/Site.css:1729]: Sometimes [we make changes in both css and less file.](https://github.com/NuGet/NuGetGallery/pull/9906#discussion_r1563263264) 
@martinrrm could please review this one?

Comment (Goodyear2017) [reply src/NuGetGallery/Content/Site.css]: I just move the css to less and compile the css file. BTW, do I need to submit .min file for the js file?

**Review by erdembayar — APPROVED** [review]

**Comment by erdembayar** [comment]
Could you please include before and after screenshots?

**Comment by Goodyear2017** [comment]
@erdembayar Please see the attached videos for before and after.
[before.webm](https://github.com/NuGet/NuGetGallery/assets/34078976/c80b747a-36f4-4c99-9a4a-5301adb980fc)
[after.webm](https://github.com/NuGet/NuGetGallery/assets/34078976/2062535d-cb13-4df9-85fc-d158248b6042)

**Comment by erdembayar** [comment]
> @erdembayar Please see the attached videos for before and after. [before.webm](https://github.com/NuGet/NuGetGallery/assets/34078976/c80b747a-36f4-4c99-9a4a-5301adb980fc) [after.webm](https://github.com/NuGet/NuGetGallery/assets/34078976/2062535d-cb13-4df9-85fc-d158248b6042)

It's very hard for me to know difference between 2 videos, just screenshots would have been better. 
Also there're `.pipelines/NuGetGallery-CI.yml` git merge conflict, you may need to rebase your branch with dev branch.

**Comment by Goodyear2017** [comment]
I will close this PR and make a new one.

**Pull request closed without merging**